extensions/interactions/AlgebraicExpressionInput/AlgebraicExpressionInput_test.py

```diff
@@ -0,0 +1,197 @@
+# coding: utf-8
+#
+# Copyright 2018 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for AlgebraicExpressionInput objects and methods defined on them."""
+
+from __future__ import annotations
+
+from core.domain import interaction_registry
+from core.domain import state_domain
+from core.tests import test_utils
+
+
+class AlgebraicExpressionInputInteractionTests(test_utils.GenericTestBase):
+
+    def test_algebric_expression_input_converted_to_proto_correctly(self):
```

Comment:
algebraic is misspelled, pretty much everywhere. Please fix all cases.

Reply:
Fixed everywhere